docs/whats_new.md

```diff
@@ -259,6 +259,8 @@ DRAFT section
 
 - Cookie now allows Partitioned cookies for [CHIPS](https://developers.google.com/privacy-sandbox/3pcd/chips) support. CHIPS (Cookies Having Independent Partitioned State) is a feature that improves privacy by allowing cookies to be partitioned by top-level site, mitigating cross-site tracking.
 
+- Introducing [CBOR](https://cbor.io/) binary encoding format for response body. CBOR is a binary data serialization format that is both compact and efficient, making it ideal for use in web applications.
```

Comment:
This only mentions `response body`, but we are also adding support for the `Request Body`. We are also adding support for CBOR to `Fiber Client` and `Binder`.

Reply:
We will create separate section for client later.

Reply:
We will create separate section for client later.

Reply:
@coderabbitai can you help here to expand this?